src/Api/KeyManagement/Models/Response/PrivateKeysResponseModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Text.Json.Serialization;
+using Bit.Core.KeyManagement.Models.Data;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.KeyManagement.Models.Response;
+
+#nullable enable
+
+/// <summary>
+/// This response model is used to return the the asymmetric encryption keys,
+/// and signature keys of an entity. This includes the private keys of the key pairs,
+/// (private key, signing key), and the public keys of the key pairs (unsigned public key,
+/// signed public key, verification key). 
+/// </summary>
+public class PrivateKeysResponseModel : ResponseModel
+{
+    [System.Diagnostics.CodeAnalysis.SetsRequiredMembersAttribute]
+    public PrivateKeysResponseModel(UserAccountKeysData accountKeys) : base("privateKeys")
+    {
+        PublicKeyEncryptionKeyPair = new PublicKeyEncryptionKeyPairModel(accountKeys.PublicKeyEncryptionKeyPairData);
+        if (accountKeys == null)
+        {
+            throw new ArgumentNullException(nameof(accountKeys));
+        }
```

Comment:
```suggestion
       ArgumentNullException.ThrowIfNull(accountKeys);
```

:pick: Completely optional on a few of these response models.